aptos-move/aptos-vm/src/block_executor/mod.rs

```diff
@@ -133,6 +145,37 @@ impl BlockAptosVM {
         state_view: &S,
         concurrency_level: usize,
     ) -> Result<Vec<TransactionOutput>, VMStatus> {
+        let addrs: HashSet<AccountAddress> = transactions
+            .iter()
+            .flat_map(|txn| {
+                if let Transaction::UserTransaction(t) = txn {
+                    Some(t.sender())
+                } else {
+                    None
+                }
+            })
+            .collect();
+
+        IO_POOL.install(|| {
```

Comment:
why not `spawn`? -- you don't need to wait for all the results to come back, right?